ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/NormalMessage.qml

```diff
@@ -122,15 +130,20 @@ Item {
             anchors.topMargin: Style.current.smallPadding
             anchors.left: parent.left
             anchors.leftMargin: chatBox.chatHorizontalPadding
+            onLoaded: {
+                chatImageContent.item.isCurrentUser = isCurrentUser
```

Comment:
I'm not sure why you need to pass it like this. `isCurrentItem` shouldn't change so you should be able to set it directly in the component no?